src/main/resources/kobiton/Server.py

```diff
@@ -4,23 +4,24 @@
 import base64
 
 errorLog = []
-def ping(params={}, headers={}):
+
+def verifyConnection(params={}, headers={}):
   try:
     request = urllib2.Request(params['url'], None, headers)
     urllib2.urlopen(request)
   except Exception as ex:
     errorLog.append("Error while connecting to {}: {}".format(params['server'], ex))
 
-ping({
-  "url": re.sub(r'\/$|\\$','',configuration.url) + '/v1/devices',
-  "server": "Kobiton"
+verifyConnection({
+  "url": re.sub(r'\/$|\\$','',configuration.apiServer) + '/v1/devices',
+  "server": "Api Server"
 }, {
   "Authorization": 'Basic %s' % base64.b64encode('%s:%s' % (configuration.username, configuration.apiKey))
 })
 
-ping({
-  "url": re.sub(r'\/$|\\$','',configuration.remoteServer) + '/ping',
-  "server": "Remote Server"
+verifyConnection({
+  "url": re.sub(r'\/$|\\$','',configuration.executorServer) + '/ping',
```

Comment:
Please add implemenetation for this endpoint to server

Reply:
It already have